Modules/_winapi.c

```diff
@@ -2806,6 +2806,11 @@ _winapi__mimetypes_read_windows_registry_impl(PyObject *module,
         if (err == ERROR_FILE_NOT_FOUND) {
```

Comment:
```suggestion
        if (err == ERROR_FILE_NOT_FOUND || err == ERROR_ACCESS_DENIED) {
```

May as well just do this instead of adding the extra case (we can't `PyErr_Set...` here anyway because the GIL is released).

Reply:
Thanks for the improvement. It's fixed now